packages/weather.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Weather < Package
+  description 'This command-line utility is intended to provide quick access to current weather conditions and forecasts.'
+  homepage 'http://fungi.yuggoth.org/weather/'
+  version '2.3'
+  source_url 'http://fungi.yuggoth.org/weather/src/weather-2.3.tar.xz'
+  source_sha256 '86148d2f1d59867f637f52558cc2a6b3280fac94df55c6e5af0ce37cc190d146'
+
+  depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"
+
+  def self.build
+    system "echo 'setpath = $HOME/.weather:#{CREW_PREFIX}/data/weather' >> weatherrc"
+    system "if [ ! -d $HOME/.weather ]; then mkdir $HOME/.weather; fi"
+    system "if [ ! -f $HOME/.weather/weatherrc ]; then cp weatherrc $HOME/.weather; fi"
```

Comment:
The weather command will add the `~/.weather` directory anyway when it caches search results.